client/bin/setup.sh

```diff
@@ -1,66 +0,0 @@
-#!/bin/bash
-set -xeuo pipefail
-
-# TODO: Just create directly in DB?
-koji add-tag build --arches=x86_64
-koji add-tag dest --arches=x86_64
-koji add-target candidate build dest
-
-koji add-pkg dest osbs-buildroot-docker --owner kojiadmin
-koji add-pkg dest docker-hello-world --owner kojiadmin
-
-koji call addBType operator-manifests
```

Comment:
Where are these koji settings replicated?

Reply:
Thanks, added in `koji-builder/bin/koji-setup.sh`